source/code-snippets/usage-examples/updateMany.ts

```diff
@@ -0,0 +1,41 @@
+import { MongoClient } from "mongodb";
+
+// Replace the uri string with your MongoDB deployment's connection string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?writeConcern=majority";
+
+const client = new MongoClient(uri);
+
+enum Rating {
+  G = "G",
+  PG = "PG",
+  PG_13 = "PG-13",
+  R = "R",
+  NR = "NOT RATED",
+}
+
+interface Movies {
+  rated: Rating;
+  random_number?: number;
```

Comment:
Is "random number" overly contrived? I don't know what this is for in the context of a Movie with ratings.

Also, do you mean "Movie" (singular) or Movies?

Reply:
Good catch, I agree "Movies" should be "Movie", I have added an item to this[ clean up tasks ticket](https://jira.mongodb.org/browse/DOCSP-18084) for making interface names singular when appropriate.